src/components/ListView.jsx

```diff
@@ -24,18 +25,25 @@ function ListView() {
       //Used to remove the snapshot listener when the component is unmounted/unsubscribed
       unsubscribe();
     };
-  }, []);
+  }, [localToken, navigate]);
 
   return (
-    <div>
-      Your Shopping List:
-      <ul>
-        {items.map((item, idx) => (
-          <li
-            key={idx}
-          >{` Name: ${item.itemName}  Frequency: ${item.frequency}`}</li>
-        ))}
-      </ul>
+    <div className="m-20">
+      {!items.length ? (
```

Comment:
@lindseyindev @djtaylor8 ,
@shelleymcq made a great observation that once the list has a least one item, going to the listView from addItemView triggers a flash of the `Welcome` component before it goes to the list items. I think what's happening here is the page is rendering before the useEffect has finished up above, so the list items haven't been registered yet. 
Here's a couple ideas...
1. One solution would be to add a loading variable in state. You could update loading after the setItems has been triggered, and add some conditional logic below to only show the component once it's done loading
2. This would be a bigger refactor, but you could load the list items in `App` and then pass them down to listItems as props. This way, the connection to firebase would only have to happen once per load of the app, as opposed to each time the list component is mounted.

Reply:
great catch and ideas to fix, I've tested first idea - it worked, and the second must work too

Reply:
I like the idea of adding a loading variable to use across the app. Is this something we should do in this PR?